src/templates/dataGridTable.jsx

```diff
@@ -39,45 +60,145 @@ const defaultProps = {
     className: undefined,
     dropShadow: false,
     fontSize: 'xsmall',
-    handle: false,
     idPrefix: 'base',
     resizableColumnWidthPercentage: undefined,
-    rowProps: undefined,
+    rowProps: () => ({
+        className: null,
+        id: null,
+        onClick: null,
+        selected: null,
+        style: null,
+    }),
     size: 'small',
+    sortable: null,
     stickyColumns: undefined,
     stretch: false,
     style: undefined,
 };
 
 class DataGridTable extends React.PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.onSortableChange = this.onSortableChange.bind(this);
+    }
+
+    onSortableChange(newData, index) {
+        const {
+            sortable,
+        } = this.props;
+
+        if (isFunction(sortable[index].onChange)) {
+            sortable[index].onChange(newData, index);
+        }
+    }
+
     render() {
         const {
             classNamePrefix,
             bleed: bleedProp,
             className,
             columns,
-            data,
+            data: dataProp,
             dropShadow,
             fontSize,
-            handle,
             id,
             idPrefix,
             rowProps,
             resizableColumnWidthPercentage,
             size,
+            sortable,
             stickyColumns,
             stretch,
             style,
         } = this.props;
-        const bleed = bleedProp ? 'very' : stretch;
-        const containerClasses = ClassNames('ui', `${classNamePrefix}_table`, className);
+
+        const isSelectable = isFunction(rowProps().onClick);
+
+        let rowKeyNum = 1;
+
+        const tableRows = (data, handle) => flatMap(data, (row) => {
+            rowKeyNum += 1;
+
+            return (
+                <DataGridTableRow
+                    classNamePrefix={classNamePrefix}
+                    columns={columns}
+                    handle={handle}
+                    id={id}
+                    idPrefix={idPrefix}
+                    isClickable={isSelectable}
+                    key={`tableBodyRow-${row.id || rowKeyNum}`}
+                    row={row}
+                    rowIndex={rowKeyNum}
+                    rowProps={rowProps(row)}
+                />
+            );
+        });
+
+        let tableBody;
         const bodyClasses = ClassNames({ [`${classNamePrefix}_drop_shadow`]: dropShadow });
-        const isSelectable =
-            !_.isUndefined(rowProps) && _.isFunction(rowProps().onClick);
+
+        if (sortable === true) {
+            tableBody = (
+                <ReactSortable
+                    className={bodyClasses}
+                    filter={`${classNamePrefix}-filter`}
+                    list={dataProp}
+                    setList={(newData) => this.onSortableChange(newData, 0)}
```

Comment:
No way to avoid the anonymous function in `render()` here, eh?
_Not a Blocker_

Reply:
Fixed.